Parsers/ASimProcessEvent/Parsers/vimProcessCreateMD4IoT.yaml

```diff
@@ -79,16 +79,19 @@ ParserQuery: |
     // --------------------------------------------------------------------------------------
     | extend
       EventDetails = todynamic(EventDetails)
+    | where EventDetails.EventType != 'EXIT'
```

Comment:
Filtering has to be done earlier as to not work on process terminate so much. As you did in terminate....

Reply:
with terminate I require inclusion, and later verified it is included in the correct place.
Here the requirement is NOT to include, so I am risking omitting cases where "EXIT" appears elsewhere

Reply:
Not if you check for the entire fragment that includes EXIT, after all it is JSON and not free text.

Reply:
After offline discussion, will not change now. 
TODO: in prefilter using the full key value segment of the EXIT value